app/mailers/sign_up_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class SignUpMailer < ApplicationMailer
+  def sign_up_email(user)
+    @user = user
+    mail(to: @user.email, subject: "Welcome to Droply")
```

Comment:
Why create the instance variable here instead of just calling it directly on the user argument passed in?